cmd/harbor/root/cmd.go

```diff
@@ -101,6 +102,7 @@ harbor help
 	root.AddCommand(
 		versionCommand(),
 		LoginCommand(),
+    member.Member(),
```

Comment:
Before committing, remember to lint your code with `'make'` or `'gofmt -s -w .'`

Reply:
thanks